repl/src/main.rs

```diff
@@ -0,0 +1,132 @@
+
+use std::path::PathBuf;
+
+use rustyline::error::ReadlineError;
+use rustyline::{Cmd, CompletionType, Config, EditMode, Editor, KeyEvent};
+
+use anyhow::Result;
+use clap::Parser;
+use directories::ProjectDirs;
+
+use hyperon::common::shared::Shared;
+
+mod metta_shim;
+use metta_shim::*;
+
+mod config_params;
+use config_params::*;
+
+mod interactive_helper;
+use interactive_helper::*;
+
+#[derive(Parser)]
+#[command(version, about)]
+struct CliArgs {
+    /// .metta files to execute.  `metta` will run in interactive mode if no files are supplied
+    files: Vec<PathBuf>,
+
+    /// Additional include directory paths
+    #[arg(short, long)]
+    include_paths: Vec<PathBuf>,
+}
+
+fn main() -> Result<()> {
+    let cli_args = CliArgs::parse();
+
+    //Config directory will be here: TODO: Document this in README.
+    // Linux: ~/.config/metta/
+    // Windows: ~\AppData\Roaming\OpenCog\metta\config\
+    // Mac: ~/Library/Application Support/org.OpenCog.metta/
+    let mut repl_params = match ProjectDirs::from("org", "OpenCog",  "metta") {
```

Comment:
I think we need to use `Hyperon` instead of `OpenCog` here, or may be even `TrueAGI`. @Necr0x0Der FYI.

Reply:
I don't have an opinion about what we put here.  But I believe the convention is that we are supposed to own / control the domain name.  We do control opencog.org, but hyperon.org and trueagi.org are for sale by domain squatters.

Reply:
We could use 'io', 'TrueAGI'

Reply:
Sound too serious for the home directory name :-) I am fine with ('io', 'TrueAGI', 'metta').

Reply:
Changed in 4660fa4